### PR TITLE
6833 Update court order validation logic

### PR DESCRIPTION
**Pull request by argush3**
*Issue #:* /bcgov/entity#6833

Note: dependent on business-schemas PR https://github.com/bcgov/business-schemas/pull/58

*Description of changes:*

* Remove temporary court order validation logic to work around issue of alteration filing schema not being triggered properly in filings endpoint previously
* Updated `test_validate_invalid_court_orders` and `test_validate_valid_court_orders ` tests to only test the `orderDate` field in the `courtOrder` object as that is now the only field that the legal api filings endpoint validates against.  The other fields are validated via the filings schema.
* Added http-client file types to gitignore

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Review by thorwolpert — CHANGES_REQUESTED** [review]
Please change to the common library.

**Comment by thorwolpert** [comment]
Maybe mark this as Draft until we settle on what to do in the schema defintions.

File: legal-api/tests/unit/services/filings/validations/alteration/test_court_order.py [hunk 1]
```diff
@@ -14,29 +14,14 @@
 """Tests to assure the Court Order is validated properly."""
 import pytest
 
+from datetime import datetime, timedelta
```
[the remark this answers is not in the document]
Comment (argush3) [reply legal-api/tests/unit/services/filings/validations/alteration/test_court_order.py]: updated to use the version you referenced

**Comment by argush3** [comment]
Ready for another review whenever you have a minute @thorwolpert.  Thanks!

**Comment by argush3** [comment]
> Maybe mark this as Draft until we settle on what to do in the schema defintions.

Just noticed this...I'll put it in draft.  Actually, I'll put the schema one in draft too

**Comment by argush3** [comment]
Talked with Thor and this will be put on hold for now. More discussion needs to happen before deciding on an approach on how schema validation will look.

**Comment by argush3** [comment]
closing for now.  we can re-open if we ever need to.

**Pull request closed without merging**